scripts/lint-style.py

```diff
@@ -14,9 +15,13 @@
 
 exceptions = []
 
-with open("scripts/style-exceptions.txt", encoding="utf-8") as f:
-    for line in f.readlines():
+SCRIPTS_DIR = Path(__file__).parent
+ROOT_DIR = SCRIPTS_DIR.parent
+
+with SCRIPTS_DIR.joinpath("style-exceptions.txt").open() as f:
```

Comment:
(This line is the only real change, the rest is just trying to ensure that doesn't screw up comparing relative and absolute paths when deciding on lint exceptions)